Lab4_Interrupts.X/clock.c

```diff
@@ -17,9 +17,9 @@ void clock_init(clock_t clock){
 }
 
 void print_clock(clock_t const clock){
-    GLCD_Value2Out(0, 2, clock->minutes);
-    GLCD_Value2Out(0, 5, clock->seconds);
-    GLCD_Value2Out(0, 8, clock->milliseconds);
+    GLCD_Value2Out_00(0, 2, clock->minutes, 2);
+    GLCD_Value2Out_00(0, 5, clock->seconds, 2);
+    GLCD_Value2Out_00(0, 8, clock->milliseconds, 2);
```

Comment:
```suggestion
    GLCD_Value2Out_00(0, 8, clock->milliseconds/100, 1);
```
No need to print unchanged digits and division by 100 might be even faster than printing the additional digit.